lib/elixir/pages/Operators.md

```diff
@@ -0,0 +1,105 @@
+`.`                                                                                      | Left to right
+`+` `-` `!` `^` `not` `~~~`                                                              | Unary
+`*` `/`                                                                                  | Left to right
+`+` `-`                                                                                  | Left to right
+`++` `--` `..` `<>`                                                                      | Right to left
+`in`                                                                                     | Left to right
+<code>&#124;></code> `<<<` `>>>` `~>>` `<<~` `~>` `<~` `<~>` <code>&lt;&#124;&gt;</code> | Left to right
+`<` `>` `<=` `>=`                                                                        | Left to right
+`==` `!=` `=~` `===` `!==`                                                               | Left to right
+`&&` `&&&` `and`                                                                         | Left to right
+<code>&#124;&#124;</code> <code>&#124;&#124;&#124;</code> `or`                           | Left to right
+`=`                                                                                      | Right to left
+`=>`                                                                                     | Right to left
+<code>&#124;</code>                                                                      | Right to left
+`::`                                                                                     | Right to left
+`when`                                                                                   | Right to left
+`<-`, `\\`                                                                               | Left to right
+`&`                                                                                      | Unary
+
+## Defining custom operators
+
+Elixir is capable of parsing a predefined set of operators; this means that it's not possible to define new operators (like one could do in Haskell, for example). However, not all operators that Elixir can parse are *used* by Elixir: for example, `+` and `||` are used by Elixir for addition and boolean *or*, but `<~>` is not used (but valid).
+
+To define an operator, you can use the usual `def*` constructs (`def`, `defp`, `defmacro`, and so on) but with a syntax similar to how the operator is used:
+
+```elixir
+defmodule MyOperators do
+  # We define ~> to return the maximum of the given two numbers,
+  # and <~ to return the minimum.
+
+  def a ~> b, do: max(a, b)
+  def a <~ b, do: min(a, b)
+end
+```
+
+To use the newly defined operators, we **have to** import the module that defines them:
+
+```elixir
+iex> import MyOperators
+iex> 1 ~> 2
+2
+iex> 1 <~ 2
+1
+```
+
+The following is a table of all the operators that Elixir is capable of parsing, but that are not used by default:
+
+* `|`
```

Comment:
Always indent bullets by two spaces. This makes them nicely aligned if there is code inside them, hence 2 spaces should be the default. :)